public/ng-views/maps.html

```diff
@@ -348,7 +349,7 @@ <h6 ng-if="!scenario[tabnav].routeId">
       </h6>
       <h6 ng-repeat="(key, variant) in variants" ng-show="variant.sel">
         <b class="badge" style="color: #{{ routes[variant.all[variant.sel].routeId][0].options.base.routeColor }}">
```

Comment:
Again, all these style inlines are custom and on the fly color updates that is why they are ok